docs/databases/swift-kuery.html

```diff
@@ -218,7 +218,7 @@ <h2 class="heading-2"><span class="blue-text">Step 2b:</span> Create a Postgres
             <p>Now we're ready to connect to our database from our Kitura server.</p>
             <div class="underline"></div>
             <h2 class="heading-2"><span class="blue-text">Step 2c:</span> Create a connection to a PostgreSQL database</h2>
-            <p>Add <a href="https://github.com/IBM-Swift/Swift-Kuery" target="_blank">Swift-Kuery</a> and <a href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL#add-dependencies" target="_blank">SwiftKueryPostgreSQL</a> to the dependencies in the `Package.swift` file. </p>
+            <p>Add <a href="https://github.com/IBM-Swift/Swift-Kuery#update-your-packageswift-file" target="_blank">Swift-Kuery</a> and <a href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL#add-dependencies" target="_blank">SwiftKueryPostgreSQL</a> to the dependencies in the `Package.swift` file. </p>
```

Comment:
The underlying Swift-Kuery README needs fixing before I can approve this change, as currently this links to instructions on how to add Swift-Kuery-PostgreSQL not Swift-Kuery (which is even more confusing below when you're following the instructiosn for MySQL).